src/main/java/edu/oakland/helper/admin/TrackData.java

```diff
@@ -123,6 +123,20 @@ public float getSpeed() {
     return points;
   }
 
+  @Override
+  public String toString() {
+    StringBuilder str = new StringBuilder();
+    if (isValid()) {
+      System.out.println("Status: " + getStatusMessage());
```

Comment:
It should always print the status, even if it is not valid.
```suggestion
    str.append("Track Data Status: " + getStatusMessage() + "\n");
    if (isValid()) {
```